scripts/create_new_branch.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# bash script to create a new branch that respects the library standard
+#### ./scripts/create_new_branch.sh "[FEAT] - Implement instrument types #14"
+#### ./scripts/create_new_branch.sh "[FEAT] Implement instrument types #14"
+#### ./scripts/create_new_branch.sh "[FEAT] - Implement instrument types#14"
+base_issue=$1
+issue_number=${base_issue##*#}
+issue="${base_issue%%#*}"
+issue="${issue//\[FEAT\] \- /}"
+issue="${issue/\[FEAT\] /}"
+issue="${issue/\[FEAT\]/}"
+issue="${issue// /\_}"
+branch_name="feat/Issue-$issue_number $issue"
```

Comment:
Usually, branch names don't contain uppercase letters.
```suggestion
branch_name="feat/issue-$issue_number $issue"
```